tensorflow/lite/kernels/CMakeLists.txt

```diff
@@ -252,8 +245,8 @@ set(TEST_WITH_EXTERNAL_MAIN_LIST
   pow_test.cc
   quant_basic_lstm_test.cc
   quantize_test.cc
-  random_standard_normal_test.cc
-  random_uniform_test.cc
+  # random_standard_normal_test.cc
```

Comment:
If they're not needed, remove them.

Reply:
Test source files for these tests are not present in `tensorflow/lite/kernels` anymore, so yes, they are not needed and are actually breaking the CMake build. So I removed them altogether.